### PR TITLE
Version 0.12.0

### PR DESCRIPTION
**Pull request by euri10**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## 0.11.9 - 2020-08-28
+
+### Added
+- Upgrade maximum h11 dependency version to 0.10 (#772) 8/28/20 54d729cc
+- Allow .json or .yaml --log-config files (#665) 8/18/20 093a1f7c
+- Added asgi dict to the lifespan scope (#754) 8/15/20 8150c3eb
```
Comment (rafalp) [inline CHANGELOG.md]:
```suggestion
- Add ASGI dict to the lifespan scope (#754) 8/15/20 8150c3eb
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+- Upgrade wsproto to 0.15.0 (#750) 8/13/20 fbce393f
+- Use optional package installs (#666) 8/10/20 5fa99a11
+
+### Changed
+- Dont set log level for root logger (#767) 8/28/20 df81b168
+
+### Fixed
+- Revert "Improve shutdown robustness when using `--reload` or multiprocessing (#620)" (#756) 8/28/20 ff4af12d
+- Fix terminate error in windows (#744) 8/27/20 dd3b842d
+- Fixes bug where --log-config disables uvicorn loggers (#512) 8/11/20 a9c37cc4
```
Comment (rafalp) [inline CHANGELOG.md]:
```suggestion
- Fix bug where --log-config disables uvicorn loggers (#512) 8/11/20 a9c37cc4
```

**Comment by hidaris** [comment]
When will it be released?

**Review by florimondmanca — COMMENTED** [review]
Given how this contains a bunch of new features, should this be a 0.12 version bump?

**Comment by euri10** [comment]
I bumped it more @florimondmanca

**Review by florimondmanca — APPROVED** [review]
Sounds good to me :)

**Pull request merged**